src/codeu/chat/util/encryption/PublicKey.java

```diff
@@ -0,0 +1,13 @@
+import java.math.BigInteger;
+
+public class PublicKey {
+    BigInteger n, e;
```

Comment:
It looks like these 2 variables can be made final since they're only assigned in the constructor.